src/components/Calculator.js

```diff
@@ -113,20 +114,23 @@ class Calculator extends React.Component {
     const { obj } = this.state;
     return (
       <div className="container">
-        <Display className="input" value={this.getDisplayValue(obj)} />
-        {lblArray.map((lbl, divIdx) => (
-          <div className="row" key={lbl[0]}>
-            {lbl.map((element, idx) => (
-              <Key
-                lbl={element}
-                cls={selectClass(idx, divIdx)}
-                key={element}
-                on
-                onSmash={() => this.handleClick(element)}
-              />
-            ))}
-          </div>
-        ))}
+        <Quote />
+        <div className="calculator">
```

Comment:
- [ ] Impressive work fetching data from `ninja API` and making use of `useState` and `useEffect` to handle the data to display it to the `UI`. However, as specified in the project requirements kindly consider importing your `Quote` component in your main component(`App`) as you did with the `Calculator` component instead of importing it in the Calculator component. Kindly consider fixing this issue to fulfill the project requirements. See below for more details on project requirements:
<img width="667" alt="Screenshot 2023-08-17 at 15 25 29" src="https://github.com/wacoo/smart-math-magicians-react/assets/107192076/335adc7b-f7b1-4d97-a1b2-9736c863bfc4">